src/workloads/query/TelemetryWithRedaction.yml

```diff
@@ -0,0 +1,234 @@
+  Type: Loader
+  Threads: 4
+  Phases:
+    OnlyActiveInPhases:
+      Active: [2]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: *Database
+        MultipleThreadsPerCollection: true
+        CollectionCount: 1
+        DocumentCount: *DocumentCount
+        BatchSize: 1000
+        Document:
+          a: &integer {^RandomInt: {min: -100, max: 100}}
+          b: *integer
+          string: &string {^RandomString: {length: 5}}
+        Indexes:
+          - keys: {a: 1, b: 1}
+          - keys: {string: 1}
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *Database
+  Phases:
+    OnlyActiveInPhases:
+      Active: [3]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+
+- Name: FindCommand
+  Type: CrudActor
+  Threads: 32
+  Database: *Database
+  Phases:
+    OnlyActiveInPhases:
+      Active: [4]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Duration: 1 minute
+        Collection: *Collection
+        Operations:
+        - OperationName: find
+          OperationCommand:
+            # This will create a find command with a random number of fields from 3 to 350, to create varying query shapes and sizes.
+            Filter:
+              {$and:
+                [{a1: 1}, {a2: 1}, {^Object: {
+                  withNEntries: {^RandomInt: {min: 1, max: 350}},
+                  havingKeys: {^RandomString: {length: 10}},
+                  andValues: {$exists: false},
+                  duplicatedKeys: skip
+                }}]
+              }
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *Database
+  Phases:
+    OnlyActiveInPhases:
+      Active: [5]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+
+- Name: FindConcurrentlyWithTelemetryWithRedaction
+  Type: CrudActor
+  Threads: 32
+  Database: *Database
+  Phases:
+    OnlyActiveInPhases:
+      Active: [6]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Duration: 1 minute
+        Collection: *Collection
+        Operations:
+        - OperationName: find
+          OperationCommand:
+            Filter:
+              {$and:
+                [{a1: 1}, {a2: 1}, {^Object: {
+                  withNEntries: {^RandomInt: {min: 1, max: 350}},
+                  havingKeys: {^RandomString: {length: 10}},
+                  andValues: {$exists: false},
+                  duplicatedKeys: skip
+                }}]
+              }
+
+# Telemetry with redaction.
+- ActorFromTemplate:
+    TemplateName: TelemetryAggregation
+    TemplateParameters:
+      Name: "TelemetryAggregationWithRedaction"
+      applyHmacToIdentifiers: true
+      OnlyActiveInPhase: 6
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *Database
+  Phases:
+    OnlyActiveInPhases:
+      Active: [7]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+
+- Name: FindConcurrentlyWithTelemetryWithoutRedaction
```

Comment:
Is this the same as the previous one? I think you could delete it and just have it active in multiple phases:
`Active: [4, 8]`

Reply:
They are the same, but combining them into 1 actor would also combine the results right? I thought we wanted to look at how `$telemetry` with and without redaction affects the performance separately? If we don't care about that then I can move them into one actor with both phases active!

Reply:
Oh that's a great point! I didn't realize/remember this. I think good to keep them separate then.